StatusBarLayer.coffee

```diff
@@ -121,13 +144,38 @@ class StatusBarLayer extends Layer
 
 		getBatteryLevel = (defaultBatteryWidth) =>
 			percentageWidth = @options.percent / 100 * defaultBatteryWidth
+			percentageWidth = Math.round(percentageWidth)
 			return percentageWidth
 
-		signalSVG = "<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 67 #{11 * getSVGFactor()}'><circle cx='5.5' cy='5.5' r='5.5' fill='#{@options.foregroundColor}' /><circle cx='19.5' cy='5.5' r='5.5' fill='#{@options.foregroundColor}' /><circle cx='33.5' cy='5.5' r='5.5' fill='#{@options.foregroundColor}' /><circle cx='47.5' cy='5.5' r='5.5' fill='#{@options.foregroundColor}' /><path d='M61.5,1A4.5,4.5,0,1,1,57,5.5,4.51,4.51,0,0,1,61.5,1m0-1A5.5,5.5,0,1,0,67,5.5,5.5,5.5,0,0,0,61.5,0Z' fill='#{@options.foregroundColor}' /></svg>"	
-		wifiSVG = "<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 25 #{18 * getSVGFactor()}'><path d='M8.59,13.63,12.5,18l3.91-4.37a5.5,5.5,0,0,0-7.82,0Zm-4-4.47,2,2.23a8.48,8.48,0,0,1,11.82,0l2-2.23a11.46,11.46,0,0,0-15.81,0ZM12.5,0A17.42,17.42,0,0,0,.6,4.7l2,2.23a14.45,14.45,0,0,1,19.81,0l2-2.23A17.42,17.42,0,0,0,12.5,0Z' fill='#{@options.foregroundColor}' /></svg>"
-		batterySVG = "<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 49 #{19 * getSVGFactor()}'><path d='M41.5,0H3.5A3.5,3.5,0,0,0,0,3.5v12A3.5,3.5,0,0,0,3.5,19h38A3.5,3.5,0,0,0,45,15.5V3.5A3.5,3.5,0,0,0,41.5,0ZM44,15.5A2.5,2.5,0,0,1,41.5,18H3.5A2.5,2.5,0,0,1,1,15.5V3.5A2.5,2.5,0,0,1,3.5,1h38A2.5,2.5,0,0,1,44,3.5Z' fill='#{@options.foregroundColor}' /><rect x='2' y='2' width='#{getBatteryLevel(41)}' height='15' rx='1.5' ry='1.5' fill='#{batteryColor}' id='batteryFill' /><path d='M46,6v7a3.28,3.28,0,0,0,3-3.5A3.28,3.28,0,0,0,46,6Z' fill='#{@options.foregroundColor}'/></svg>"
-		batterySVG3x = "<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 73 #{29 * getSVGFactor()}'><path d='M62,0H5A5,5,0,0,0,0,5V24a5,5,0,0,0,5,5H62a5,5,0,0,0,5-5V5A5,5,0,0,0,62,0Zm4,24a4,4,0,0,1-4,4H5a4,4,0,0,1-4-4V5A4,4,0,0,1,5,1H62a4,4,0,0,1,4,4Z' fill='#{@options.foregroundColor}' /><rect x='2' y='2' width='#{getBatteryLevel(63)}' height='25' rx='3' ry='3' fill='#{batteryColor}' id='batteryFill' /><path d='M69,10.06v9.89A4.82,4.82,0,0,0,73,15,4.82,4.82,0,0,0,69,10.06Z' fill='#{@options.foregroundColor}' /></svg>"
-		powerSVG = "<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 11 #{19 * getSVGFactor()}'><polygon points='11 7.5 5.86 7.5 8 0 0 10.5 4.96 10.5 2 19 11 7.5' fill='#{@options.foregroundColor}' /></svg>"
+
+		signal_v10_2x = "<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 34 16'><circle cx='2.75' cy='2.75' r='2.75' fill='#{@options.foregroundColor}' /><circle cx='9.75' cy='2.75' r='2.75' fill='#{@options.foregroundColor}' /><circle cx='16.75' cy='2.75' r='2.75' fill='#{@options.foregroundColor}' /><circle cx='23.75' cy='2.75' r='2.75' fill='#{@options.foregroundColor}' /><circle cx='30.75' cy='2.75' r='2.5' stroke='#{@options.foregroundColor}' stroke-width='0.5' fill-opacity='0' class='stroked' /></svg>"
```

Comment:
Generally we have an lint rule that requires camelCase for all variables names but in this context since we do not have linting privileges for coffeescript I think that is fine